Tasks/Korotkevich_Grigory/Task_2/Task2.py

```diff
@@ -0,0 +1,98 @@
+
+'''Реализовать:
+1.текстовый вывод сегодняшнего времени
+2.текстовый вывод времени, ведомого с консоли (чч:мм).
+(дать возможность выбора режима работы программы)
+
+Для получения свободного времени можно использовать модуль datetime.
+
+min == 0: такое-то значение часа ровно
+мин < 30: столько-то минут следующих часов
+мин == 30: половина такого-то
+мин > 30 и мин < 45 столько-то минут следующих часов
+мин >= 45 без мин такого-то
+
+21:16
+шестнадцать минут десятого'''
+
+
+
+
+
+import datetime as dt
+
+minutes_count = {
+1: ['одна', 'одной'], 2: ['две', 'двух'], 3: ['три', 'трех'],
+4: ['четыре', 'четырёх'], 5: ['пять', 'пяти'], 6: ['шесть', 'шести'],
+7: ['семь', 'семи'], 8: ['восемь', 'восьми'], 9: ['девять', 'девяти'],
+10: ['десять', 'десяти'], 11: ['одинадцать', 'одинадцати'], 
+12: ['двенадцать', 'двенадцати'], 13: ['тринадцать', 'тринадцати'],
+14: ['четырнадцать', 'четарнадцати'], 15: ['пятнадцать', 'пятнадцати'],
+16: ['шестнадцать'], 17: ['семнадцать'], 18: ['восемнадцать'], 19: ['девятнадцать'],
+20: ['двадцать'], 30: ['тридцать'], 40: ['сорок'], 50: ['пятьдесят']
+}
+
+counting_hours = {
+0: ['ноль'],1: ['час', 'первого'],2: ['два', 'второго'],3: ['три', 'третьего'],4: ['четыре', 'четвертого'],5: ['пять', 'пятого'],
+6: ['шесть', 'шестого'],7: ['семь', 'седьмого'],8: ['восемь', 'восьмого'],9: ['девять', 'девятого'],10: ['десять', 'десятого'],
+11: ['одинадцать', 'одинадцатого'],12: ['двенадцать', 'двенадцатого'],
+}
+
+
+player = int(input('Enter 1 if you want to know the current time!\n'
+                   'Enter 2 and you will need to manually enter the time in hh:mm format!\n'))
+
+
+if player == 1:
+    time = dt.datetime.now().time()
+    time = time.strftime('%H:%M')
+
+elif player == 2:
+    time = input()
```

Comment:
so first of all, show user more explicitly that you want they to input data

Reply:
ok, I'll take it into account